src/widget/container/mod.rs

```diff
@@ -0,0 +1,156 @@
+mod alignment;
+mod border;
+
+pub use alignment::Alignment;
+pub use border::Border;
+
+use super::{IntrinsicSize, Widget};
+use alloc::{boxed::Box, vec::Vec};
```

Comment:
Oh weird, this is how you use heap-allocated things in `no_std` world?

Reply:
Yeah, and you also need to configure a global allocator in your “main” file. I don’t need to do that in the examples because that are not actually `no_std` apps.